packages/keystone/lib/Keystone/index.js

```diff
@@ -419,7 +419,7 @@ module.exports = class Keystone {
     return mergeRelationships(createdItems, createdRelationships);
   }
 
-  async prepare({ port = DEFAULT_PORT, dev = false, apps = [], distDir } = {}) {
+  async prepare({ dev = false, apps = [], distDir } = {}) {
```

Comment:
Can also remove the `DEFAULT_PORT` import from the top of this file.

Reply:
Done